pkg/apis/serving/v1alpha2/framework_tensorflow.go

```diff
@@ -50,12 +53,24 @@ func (t *TensorflowSpec) GetContainer(modelName string, parallelism int, config
 		"--model_base_path=" + constants.DefaultModelLocalMountPath,
 	}
 
+	// Define TensorFlow Serving gRPC port.
+	TensorflowServingGRPCPortInt, err := strconv.Atoi(TensorflowServingGRPCPort)
+	if err == nil {
```

Comment:
should be `err != nil` or just define the constant as int32

Reply:
done